File: sentimental-mario-less/mario.py
from cs50 import get_int

while True:

    try:
        height = get_int("Height: ")

        if height > 8 or height < 1:
            pass
        else:
            break
    except:
        pass

for i in range(height):

    spaces = height - 1
    bricks = i + 1

    for i in range(spaces):
        print(f" ", end="")

    for i in range(bricks):
        print(f"#", end="")

    height -= 1
    print("")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    void buildpyramid(int);

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    buildpyramid(height);
}

void buildpyramid(int z)
{
    int i;
    int j;
    int k;
    int spaces;
    int bricks;

    for (i = z; i > 0; i--)
    {
        spaces = i - 1;
        bricks = z - spaces;

        for (j = spaces; j > 0; j--)
        {
            printf(" ");
        }

        for (k = bricks; k > 0; k--)
        {
            printf("#");
        }

        printf("\n");
    }
}
